pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/db/dao/mercury/ClinicalOrderDaoTest.java

```diff
@@ -64,7 +64,7 @@ public class ClinicalOrderDaoTest extends DbTxnBaseTest {
 
     private static int oncHistoryDetailId = -1;
 
-    public static TestKitUtil testKitUtil;
```

Comment:
That's basically the only change in this class, and below is only the name changed for pepperKitUtil